cmd/kubebuilder/version1/version1.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version1
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/kubernetes-sigs/kubebuilder/cmd/kubebuilder/update"
+	"github.com/kubernetes-sigs/kubebuilder/cmd/kubebuilder/version"
+)
+
+func AddVersion1Cmd(cmd *cobra.Command) {
+	AddAPICommand(cmd)
+	update.AddUpdate(cmd)
```

Comment:
version0 update may not work for version1. I think there is no update command in controller-tools. We will have add one in controller-tools and then add it here.